tailwind-showcases/src/TailwindShowcases.tsx

```diff
@@ -43,6 +43,8 @@ const getClassSuffixes = (theme: Record<string, any>) => ({
   fontWeight: extractClassSuffixes('fontWeight', theme),
   letterSpacing: extractClassSuffixes('letterSpacing', theme),
   lineHeight: extractClassSuffixes('lineHeight', theme),
+  textDecoration: ['underline', 'line-through', 'no-underline'],
```

Comment:
Let's remove this.

Reply:
Yes, I'm missing a way to discard changes in the DS when viewing the diff.